app/assets/stylesheets/petitions/admin/_temporary_tab_nav.scss

```diff
@@ -0,0 +1,16 @@
+.admin {
```

Comment:
It's a small namespace for any instance where we are overwriting public styles. This may change but it helps reduce the likelihood someone will make a change that will effect the public site.